docs/Commands.html

```diff
@@ -89,25 +89,23 @@ <h3>Command <code>createBashCompletionScript</code></h3>
 
 <h3>Command <code>explore</code></h3>
 <p>
-This command starts Shasta in a mode in which it behaves as an
+This command starts Shasta in a mode that behaves as an
```

Comment:
Similar comment to a similar change yesterday. The new wording does not sound right to me.

Reply:
it is correct :)